### PR TITLE
WRLDAEMS-62: Footer feedback

### PR DESCRIPTION
**Pull request by adobe-rpal**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://WRLDAEMS-62--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,450 +1,521 @@
...
+	}
 }
+
+@media only screen and (width > 1024px) {
+	.ft-social {
+		grid-template-columns: 1fr 1fr 2fr;
+		grid-column-gap: 0;
+	}
+
+	html[dir="rtl"] .ft-main-item:last-child {
```
Comment (thomsebastin) [inline blocks/footer/footer.css:518]: @adobe-rpal Could you please fix the indentation of this file.

Comment (adobe-rpal) [reply blocks/footer/footer.css:518]: Done

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -1,450 +1,521 @@
...
+	background-color: var(--primary-blue-100);
+  }
+  
+  footer .block {
+	max-width: var(--screen-width);
+	margin: auto;
+	padding: 0 var(--section-gutter-space);
+  }
+  
+  footer a {
```
Comment (thomsebastin) [inline blocks/footer/footer.css:12]: @adobe-rpal Does this indentation look correct to you?

File: blocks/footer/footer.css [hunk 3]
```diff
@@ -1,450 +1,521 @@
...
-      }
-
-      & a {
-        margin: 0;
-      }
-    }
-  }
-
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (thomsebastin) [inline blocks/footer/footer.css:162]: `repeat(4, 1fr)`

File: blocks/footer/footer.css [hunk 4]
```diff
@@ -1,450 +1,521 @@
...
+  
+  .ft-main .ft-main-item {
+	padding-right: 28px;
+	min-width: 20%;
+  }
+  
+  .ft-main .ft-main-item a:not(.button) {
+	display: inline-block;
+	font-size: 16px;
+	line-height: 22.4px;
```
Comment (thomsebastin) [inline blocks/footer/footer.css:175]: round the value.

File: blocks/footer/footer.css [hunk 5]
```diff
@@ -1,450 +1,521 @@
...
+  
+  .ft-main .ft-main-item ul li a.button {
+	margin: 0;
+  }
+  
+  .ft-main .ft-main-item p {
+	font-size: 26px;
+	text-transform: uppercase;
+	font-weight: 400;
+	line-height: 36.4px;
```
Comment (thomsebastin) [inline blocks/footer/footer.css:186]: round the value

File: blocks/footer/footer.css (hunk 1 above)
Comment (thomsebastin) [reply blocks/footer/footer.css:518]: Not fixed yet. Check the diff.

**Review by thomsebastin — CHANGES_REQUESTED** [review]

**Comment by adobe-rpal** [comment]
All changes are done. Kindly verify.

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**